Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:lts-alpine
+
+RUN apk update
+
+WORKDIR /app
+
+COPY . ./
+
+RUN yarn
+
+RUN yarn build
+
```

i
```suggestion
****
```
https://matrix.to/#/@matthew:matrix.org
192.168.0.1

---

Thank you for your reply, but I don't get what you're trying to say, do I need to have further modification on the Dockerfile? (e.g. https://github.com/henryclw/matrix.to/blob/dev-docker/Dockerfile)